src/main/java/org/reaktivity/nukleus/mqtt/internal/stream/MqttServerFactory.java

```diff
@@ -586,6 +594,61 @@ private void onMqttUnsubscribe(
         private void onMqttPublish(
             MqttPublishFW publish)
         {
+            if (publish != null)
+            {
```

Comment:
Do we need to test for `null` here or is `publish` guaranteed to always be `non-null`?